core/src/wallet/stellar/database/StellarLikeAccountDatabaseHelper.cpp

```diff
@@ -122,8 +122,8 @@ namespace ledger {
                                                              int32_t accountIndex, const stellar::Account &in) {
             auto accountUid = AccountDatabaseHelper::createAccountUid(walletUid, accountIndex);
             sql << "INSERT INTO stellar_accounts VALUES (:uid, :wallet_uid, :idx, :address, :sequence, :count)",
-                    use(accountUid), use(walletUid), use(accountIndex), use(in.accountId), use(in.sequence),
-                    use(in.subentryCount);
+                    use(accountUid), use(walletUid), use(accountIndex), use(in.accountId),
+                    use(in.subentryCount), use(in.sequence);
```

Comment:
The `:count` is the last value but you provide the sequence instead... The previous code looks like the correct way to create the account UID

Reply:
The names are not really important if you don’t name them in the `soci::use`. The fix here is to swap the names in the SQL query because otherwise it’s confusing. But that code will do the right thing.

Reply:
The label are not correct, I correct them. The new order is the good one. regarding the account table schema.